services/workflows-service/scripts/seed.ts

```diff
@@ -146,19 +146,45 @@ async function seed() {
     }),
   });
 
+  const endUser1 = await client.endUser.create({
+    data: generateEndUser({
+      projectId: project1.id,
+    }),
+  });
+
+  const endUser2 = await client.endUser.create({
+    data: generateEndUser({
+      projectId: project1.id,
+    }),
+  });
+
   const ids1 = await generateTransactions(client, {
     projectId: project1.id,
-    businessId: business1.id,
+    business: business1,
+    endUser: endUser1,
   });
   const ids2 = await generateTransactions(client, {
     projectId: project1.id,
-    businessId: business2.id,
+    business: business2,
+    endUser: endUser1,
+  });
+  const ids3 = await generateTransactions(client, {
+    projectId: project1.id,
+    business: business2,
+    endUser: endUser1,
+  });
+  const ids4 = await generateTransactions(client, {
+    projectId: project1.id,
+    business: business2,
+    endUser: endUser2,
```

Comment:
Added some variety to our transactions seed:
1. Transaction with both originator & beneficiary that can be either business or end user
2. Transaction with originator only
3. Transaction with beneficiary only